pandas/core/arrays/_mixins.py

```diff
@@ -139,7 +138,6 @@ def repeat(self: _T, repeats, axis=None) -> _T:
         --------
         numpy.ndarray.repeat
         """
-        nv.validate_repeat(tuple(), dict(axis=axis))
```

Comment:
why is this no longer needed? or unrelated?

Reply:
can you comment here @jbrockmendel

Reply:
In IntervalArray.repeat we call `self._combined.repeat(repeats, 0)`, which when _combined is DTA/TDA gets here with `axis=0`

Reply:
Since there is still discussion about whether `_combined` is meant to stay (https://github.com/pandas-dev/pandas/pull/37047), can you leave out this clean-up from this PR? 

(unless the clean-up is not tied to having `_combined`, but from your comment it seems so)